code/src/components/categorySection/CategorySection.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { IntlProvider } from "react-intl";
+import { render, screen } from "@testing-library/react";
+import CategorySection from "@/components/categorySection/CategorySection";
+import paragraphs from "@/shared/i18n/common-messages/en.json";
+import categorySectionElements from "@/components/categorySection/CategorySection.constants";
```

Comment:
Here can remove `IntProvider` with `paragraphs` imports
And also remove wrapper for your `CategorySection` it'll work for your case

Reply:
Done